latency.go

```diff
@@ -1,87 +1,90 @@
 package main
 
 import (
-	"fmt"
 	"io/ioutil"
-	"math/rand"
 	"net/http"
 	"strconv"
-	"time"
 
+	log "github.com/sirupsen/logrus"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 )
 
-var Latency = make(map[string]float64)
+var (
+	latencyReadChan  = make(chan map[string]float64)
+	latencyWriteChan = make(chan map[string]float64)
+)
 
-type pvLatdata struct {
-	readLatency  float64
-	writeLatency float64
-}
+var (
+	latencyReadQuery  = "increase(openebs_read_time[5m])/1000000"
+	latencyWriteQuery = "increase(openebs_write_time[5m])/1000000"
+)
 
-func getLatValues(urlpassed string, query string) {
-	res, err := http.Get(urlpassed)
+func getLatencyValues(query string) {
```

Comment:
Is there a way to combine the code? I see these methods are similar to each. You can tie these methods to the struct which has these parameters in it and method to work on those parameter.